developer/testscripts/aatest_ds000114_fmri_localpipeline.m

```diff
@@ -0,0 +1,126 @@
+function aatest_ds000114_fmri_localpipeline(parameterfile, deleteprevious,wheretoprocess)
+
+% This script runs a basic modeling pipeline (preprocessing through secondlevel)
+% using ds000114 motor and line bisection tasks. The data should be dowloaded 
+% prior to running the script
+
+% -------------------------------------------------------------------------
+% init
+% -------------------------------------------------------------------------
+
+aap = aa_test_inittest(mfilename('fullpath'), parameterfile, deleteprevious, wheretoprocess);
+aap = aarecipe(['$AAHOME/developer/testscripts/aatest_ds000114_fmri_localpipeline_prep.xml']);
+
+% -------------------------------------------------------------------------
+% First pre-process the data.
+% -------------------------------------------------------------------------
+
+aap.options.autoidentifystructural_choosefirst = 1;
+aap.options.autoidentifystructural_chooselast = 0;
+
+aap.acq_details.root = '/home/runner/projects';
+aap.directory_conventions.analysisid = 'aatest_ds000114_fmri_local_prep';
+prep_dir = 'aatest_ds000114_fmri_local_prep';
```

Comment:
You do not set `aap.acq_details.root` and `aap.directory_conventions.analysisid` in CI. They are set automatically by ` aa_test_inittest`.
You can, however, use them:
````matlab
prep_dir = fullfile(aap.acq_details.root,aap.directory_conventions.analysisid);
````